externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/schema/RowSetHelper.scala

```diff
@@ -43,7 +51,7 @@ abstract class RowSetHelper {
       val columnSize = row.size
       var j = 0
       while (j < columnSize) {
-        val columnValue = toTColumnValue(j, row, columns)
+        val columnValue = toTColumnValue(row, j, columns)
```

Comment:
No need to change the signature of method `toTColumnValue`. The param `ordinal` is also used in first place in the implementation.